hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/map/UpdatingEntryProcessor.java

```diff
@@ -119,15 +111,13 @@ public void writeData(ObjectDataOutput out) throws IOException {
         out.writeObject(rowProjectorSupplier);
         out.writeObject(valueProjectorSupplier);
         out.writeObject(arguments);
-        ImdgUtil.writeSubject(out, subject);
```

Comment:
I was thinking about sending `subject` but ignoring it during deserialization. However, it seems that it will not make sense even in the future, because sending `subject` in `UpdatingEntryProcessor` data is inherently insecure. It should be injected from the outside.

Reply:
hmm, in case of member protocol we can trust the content of  serialized data, we cannot do that in case of client protocol. But currently serialization does not distinguish them - I do not see any way to know via `ObjectDataInput` where the data comes from.
But if it were possible, it would be a nice solution as `UpdatingEntryProcessor` should never be sent by client, but currently it is possible because it can be deserialized.